src/base/utils/string.cpp

```diff
@@ -119,23 +57,25 @@ Utils::String::NaturalCompare::NaturalCompare()
 #endif
 }
 
-bool Utils::String::NaturalCompare::operator()(const QString &l, const QString &r)
+bool Utils::String::NaturalCompare::operator()(const QString &left, const QString &right)
 {
+    // case-insensitive comparison
 #if (QT_VERSION >= QT_VERSION_CHECK(5, 2, 0))
 #if defined(Q_OS_WIN)
     // Without ICU library, QCollator doesn't support `setNumericMode(true)` on OS older than Win7
     if(QSysInfo::windowsVersion() < QSysInfo::WV_WINDOWS7)
         return lessThan(l, r);
```

Comment:
Thanks for catching this.
The next time please don't post the same issue multiple times...